.github/workflows/integration_test.yaml

```diff
@@ -7,6 +7,9 @@ jobs:
   integration-tests:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
+    permissions:
+      contents: read
+      packages: write
```

Comment:
Is this still needed? I think they fixed the settings

Reply:
https://github.com/canonical/operator-workflows/blob/main/.github/workflows/build_rocks.yaml#L82

I think it is required due to this line, otherwise it would throw an invalid workflow error (example to the run below). 
https://github.com/canonical/jenkins-k8s-operator/actions/runs/7169188246

I'm not sure if this is also fixed by enterprise level permissions, i'll remove and try running it as well.

Reply:
Yeh I saw somewhere that it should be fixed now